config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root to: "cities#index"
-  resources :cities, only: [:index, :show] do
-    resources :categories, only: [:show]
+  resources :cities, only: [:index] do
+    resources :categories, only: [:show, :index]
   end
+
+  resources :cart_categories, only: [:create]
+  get '/cart', to: 'cart_categories#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.